af-plans/juiceshop-selenium-auth/README.md

```diff
@@ -15,7 +15,6 @@ To run this plan you need to:
 * Add a user to Juice Shop with the credentials:
   * username: test@test.com
   * password: test123
-* Edit all of the automation plans and shell scripts to replace `/full/path/` to be the local full path of your scripts
```

Comment:
The shell scripts part should be kept.

There are two entries under `Plans` that can be removed (relative paths and the blog post, guess it can also link to it).